src/main/java/com/ibm/disni/rdma/verbs/impl/NatPostSendCall.java

```diff
@@ -86,6 +86,7 @@ public void set(IbvQP qp, List<IbvSendWR> wrList) {
 		}
 
 		if (cmd != null){
+			assert cmd.size() >= size;
```

Comment:
I'm in favor of removing the set method completely. We can pass the arguments to the constructor. The size of an SVC object should be fixed once constructor (it can still be modified, e.g., offset etc, but the size remains so don't need to the set method).